### PR TITLE
Create simple render view

### PR DESCRIPTION
**Pull request by serhii73**

### CODE REVIEW

**Comment by serhii73** [comment]
@webknjaz I renamed this PR. Good now?

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -28,13 +28,13 @@
     - --output-format=colorized
     - |-
         --init-hook=
-        import os, platform, shutil, sys;
+        import os, platform, sys;
```
Comment (webknjaz) [inline .pre-commit-config.yaml]: This is unrelated.

Comment (serhii73) [reply .pre-commit-config.yaml]: Sorry, @webknjaz , I need to delete it?

Comment (webknjaz) [reply .pre-commit-config.yaml]: You don't not introduce unrelated changes in PR.
